core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java

```diff
@@ -24,6 +24,13 @@
   String customerSignupPath = "/V1/customers";
   String customerSigninPath = "/V1/integration/customer/token";
   String customerProfilePath = "/V1/customers/me";
+    String customerCreateCartPath = "/V1/carts/mine";
+    String customerAddToCartPath = "/V1/carts/mine/items";
```

Comment:
@DeepaliAgrwl : why are there 2 variables with same value customerAddToCartPath and customerAddToCartPath

Reply:
@rishabh-behl one it for AddToCart and One Is for CreateCart. Variable names are also different.